documentation.html

```diff
@@ -696,11 +696,274 @@ <h3>Rs.4900 <Span>Rs.9999</Span></h3>
 
             </div>
 
-            
+            <hr>
 
+            <!-- Image -->
             
+            <div id="image">
+                <h1 class="h1-style">Image</h1>
+                <p class="p-style">Image components are used to display images on the websites.It can be of various
+                    styles but most prominent ones are squares & circle with responsiveness as an important property.
+                </p>
+
+                <!-- square-img -->
+
+                <div class="card1">
+                    <h2 class="h2-style">Square Image</h2>
+                    <p class="p-style">Square Images are the square shaped, with a default dimension of 150px X 150px.
+                        Use class of <span>square-img</span> to give square shape on an image. 
+                    </p>
+
+                    <div class="display flex-row">
+                        
+                            <img class="square-img" src="./images/nike.png" alt="sneakers">
+                            <img class="square-img" src="./images/sample1.jpg" alt="alt-image">
+                            <img class="square-img" src="./images/sample2.jpg" alt="alt-image">
+
+
+                    </div>
+
+                    <div class="flex-row code-snippet">
+                        <iframe
+                            src="https://carbon.now.sh/embed?bg=rgba%28255%2C255%2C255%2C1%29&t=cobalt&wt=none&l=auto&width=873.5&ds=false&dsyoff=20px&dsblur=68px&wc=true&wa=false&pv=0px&ph=0px&ln=false&fl=1&fm=Fira+Code&fs=14px&lh=152%25&si=false&es=2x&wm=false&code=%2509%2509%2509%253Cimg%2520class%253D%2522square-img%2522%2520src%253D%2522.%252Fimages%252Fnike.png%2522%2520alt%253D%2522sneakers%2522%253E%250A%2509%2509%2509%253Cimg%2520class%253D%2522square-img%2522%2520src%253D%2522.%252Fimages%252Fsample1.jpg%2522%2520alt%253D%2522alt-image%2522%253E%250A%2509%2509%2509%253Cimg%2520class%253D%2522square-img%2522%2520src%253D%2522.%252Fimages%252Fsample2.jpg%2522%2520alt%253D%2522alt-image%2522%253E"
+                            style="width: 873px; height: 122px; border:0; transform: scale(1); overflow:hidden;"
```

Comment:
do not use inline styles.

Reply:
this is coming from code snippet tool so ignoring it.